scraper/PSQL.py

```diff
@@ -24,16 +22,20 @@ def insertToApplication(db, url, followers, appName, hash):
     return application_id
 
 
-def insertToPackages(db, name):
-    cur = db.cursor()
+def insertToPackages(cur, name, downloads_last_month, categories, modified):
+
+    # Reformat the category array to a string literal for PostgreSQL
+    categoryString = None
+    if categories and len(categories) > 0:
+        categoryString = str(categories).replace("'", "").replace("[", "{").replace("]", "}")
+
     cur.execute(
-        "INSERT INTO packages (name, retrieved) VALUES (%s, %s) ON CONFLICT(name) DO UPDATE SET name=EXCLUDED.name RETURNING id;",
-        (name, datetime.datetime.now()))
+        "INSERT INTO packages (name, downloads_last_month, categories, modified, retrieved) VALUES (%s, %s, %s, %s, %s) ON CONFLICT(name) DO UPDATE SET (name, downloads_last_month, categories, modified) = (EXCLUDED.name, EXCLUDED.downloads_last_month, EXCLUDED.categories, EXCLUDED.modified) RETURNING id;",
```

Comment:
This is just my personal suggestion after reading this line. Can we put all the queries and commands like this into a configuration file and read all of them including constants together when we reboot the application. It might be easy to manage and maintain. Also, we do not need to type in all the parameters(eg. token or db_name) when we run from CLI.

Reply:
Love this idea, I'll factor these out into separate variables.